### PR TITLE
CLN: Old string formatting: .format() -> f

### PR DESCRIPTION
**Pull request by DorAmram**
- [ ] ref #29547
- [ ] tests added / passed
- [ ] passes `black pandas`
- [ ] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/_version.py [hunk 1]
```diff
@@ -101,10 +101,8 @@ def versions_from_parentdir(parentdir_prefix, root, verbose):
     if not dirname.startswith(parentdir_prefix):
         if verbose:
             print(
-                "guessing rootdir is '{root}', but '{dirname}' "
-                "doesn't start with prefix '{parentdir_prefix}'".format(
-                    root=root, dirname=dirname, parentdir_prefix=parentdir_prefix
-                )
+                f"""guessing rootdir is '{root}', but '{dirname}'
+                doesn't start with prefix '{parentdir_prefix}'"""
```
Comment (topper-123) [inline pandas/_version.py]: don't use triple-quoted strings, it's not the same as two single-quoted strings.

**Review by topper-123 — CHANGES_REQUESTED** [review]
Looks good, a single issue.

**Comment by alimcmaster1** [comment]
Hmm I think this file is autogenerated as per @jbrockmendel comments here https://github.com/pandas-dev/pandas/pull/29518#discussion_r344826266

**Comment by jreback** [comment]
yeah we can leave this one alone, do we have a check for .format usage in checks? (prob not yet)

**Comment by alimcmaster1** [comment]
> yeah we can leave this one alone, do we have a check for .format usage in checks? (prob not yet)

Not yet there are still 200+ files to tackle. 
`grep -l -R -e '%s' -e '%d' -e '\.format(' --include=*.{py,pyx} pandas/`

If we want to add a check now we can do - and users remove the file from exclude list once done?
(Would help confirm all instances of .format have been removed in the PR for a file)

**Comment by jbrockmendel** [comment]
grepping for `.format(` will be complicated by the fact that we have 6 places where we `def format\(`.  But we can worry about that later.  This PR is non-actionable, closing.

**Pull request closed without merging**